meson.build

```diff
@@ -2,7 +2,7 @@ project('libratbag', 'c',
 	version : '0.14',
 	license : 'MIT/Expat',
 	default_options : [ 'c_std=gnu99', 'warning_level=2' ],
-	meson_version : '>= 0.40.0')
+	meson_version : '>= 0.50.0')
```

Comment:
Ignore the warning, they are taken care of. Please revert this.

Reply:
Reverted, makes sense.